en/integrations/plugin-packs/procedures/hardware-devices-polycom-dma-snmp.md

```diff
@@ -0,0 +1,215 @@
+---
+id: hardware-devices-polycom-dma-snmp
+title: Polycom DMA SNMP
+---
+
+## Overview
+
+The Polycom RealPresence Distributed Media Application (DMA) is a unique network-based 
+virtualization application that provides conference manager and call server functionality 
+for managing and distributing calls across collaboration networks. 
+
+The RealPresence DMA system is available in an Appliance Edition and a Virtual Edition 
+(packaged as software only) that can be deployed on VMware, Hyper-V, KVM, Amazon AWS 
+or Microsoft Azure cloud.
+
+## Plugin-Pack assets
+
+### Monitored objects
+
+* DMA Devices (hardware & software)
+* DMA Clusters
+* Attached Clusters, Servers & Device registrations status
+
+### Collected metrics
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Alerts-->
+
+| Metric name             | Description        | Unit  |
+| :---------------------- | :----------------- | :---- |
+| dma.alerts.total.count  | Number of alerts.  | Count |
+
+Specify through the `--warning-status` and `--critical-status` options which severities

```

Comment:
That doesn't "brake" the "DOCUSAURUS_CODE_TABS" ?

Reply:
No, this should be ok as for https://raw.githubusercontent.com/centreon/centreon-documentation/master/en/integrations/plugin-packs/procedures/operatingsystems-linux-snmp.md for example